src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -310,11 +310,17 @@ public void error(SAXParseException e) throws SAXException {
             for (int j = 0; j < features.getLength(); j++) {
                 String content = features.item(j).getTextContent();
                 if (content != null) {
-                    if (content.contains(":")) {
-                        debug("The feature " + content + " in the server.xml file is a user feature and its installation will be skipped.");
-                    } else {
-                        result.add(content.trim().toLowerCase());
-                    }
+                	content = content.trim();
+                	if (content.contains(":")) {
+                		String[] contentsplit = content.split(":");
+                		if (contentsplit.length > 2) {
+                			debug("improper feature name format, this features will be skipped: " + content);
```

Comment:
"The format of feature " + content + " in the server.xml is not valid and its installation will be skipped."